system-int-test/test/cmr/system_int_test/search/collection_search_format_test.clj

```diff
@@ -624,14 +624,20 @@
         (let [results (search/find-concepts-kml :collection {:concept-id (:concept-id coll1)})]
           (dk/assert-collection-kml-results-match [coll1] results))))
 
-    (testing "csv is not supported"
-      (is (= {:errors ["The mime type [text/csv] is not supported for collections."],
-              :status 400}
-             (search/find-concepts-csv :collection {})))
+    (testing "csv"
+      (let [response (search/find-concepts-csv :collection {})]
+        (is (= (:status response) 200))
+        (is (string/starts-with?
```

Comment:
I found that the search results here were changing between test runs, hence `starts-with`

Reply:
We have to check on the granule csv values in the test. Try add sort parameter to make the granules come back in a predictable order, so that we can verify the values.